Cartfile

```diff
@@ -0,0 +1,2 @@
+github "tonymillion/Reachability" "master"
+github "square/SocketRocket"
```

Comment:
It gets fixed in the Cartfile.resolved file I think https://github.com/pivotal-ivan-rodriguez/libPusher/blob/ios-framework/Cartfile.resolved#L2